youtube_dl/extractor/vier.py

```diff
@@ -105,13 +126,13 @@ def _real_extract(self, url):
         entries = []
         for current_page_id in itertools.count(start_page):
             current_page = self._download_webpage(
-                'http://www.vier.be/%s/videos?page=%d' % (program, current_page_id),
+                'http://www.%s.be/%s/videos?page=%d' % (site, program, current_page_id),
                 program,
                 'Downloading page %d' % (current_page_id + 1))
             page_entries = [
-                self.url_result('http://www.vier.be' + video_url, 'Vier')
+                self.url_result('http://www.' + site + '.be' + video_url, 'Vier')
                 for video_url in re.findall(
-                    r'<h3><a href="(/[^/]+/videos/[^/]+(?:/\d+)?)">', current_page)]
```

Comment:
You are right, it does seem to match the videos twice. I did not catch that issue.

vijf.be seems to have a `<h2>` tag in front of the video URL though. Is it warranted to change the regex to `<h[23]>`, or is this the point where I should build a separate extractor?